src/mongo/db/audit/audit.cpp

```diff
@@ -247,14 +364,22 @@ namespace audit {
             // coverage on the code that generates audit log objects.
             DEV {
                 log() << "Initializing dev null audit..." << std::endl;
-                _setGlobalAuditLog(new VoidAuditLog());
+                _setGlobalAuditLog(new VoidAuditLog(fromjson(auditOptions.filter)));
```

Comment:
I don't think there are explicit tests which are testing VoidAuditLog. As it is said in the comments VoidAuditLog exists to increase code coverage. And I believe this patch does not change coverage at least when filter is not specified on the command line.
By default filter is empty that is no messages are filtered out.
So I think this is harmless.